modules/API/pages/gsql-rest/gsql-rest-intro.adoc

```diff
@@ -5,8 +5,8 @@
 
 This user guide provides information on how to engage with our REST APIs: the introduction section explains how to send requests, pass parameters, and format request bodies, while xref:gsql-rest/gsql-endpoints.adoc[] describes in detail each endpoint and its input and output.
 
-To submit a request, send an HTTP request to the NGINX server or the GSQL server.
-By default, the GSQL server listens for requests at port 8123 and the NGINX server listens on port 14240.
+To submit a request, send an HTTP request to the NGINX server with the port number 14240.
+By default, the NGINX server listens on port 14240 and forward request to correct service, such as GSQL, RESTPP and GUI.
```

Comment:
port 8123 is internal only (only accessible among cluster nodes), not open to public.